source/composer.apib

```diff
@@ -6,6 +6,20 @@ The composition of API Blueprint is performed as it is provided by the [Matter C
 
 #### Input Media Types
 
+##### API Elements
+
+API Elements is supported, you can send an [API Elements API
+Category](http://api-elements.readthedocs.io/en/latest/element-definitions/#category-base-api-element)
+to the composer using the `application/vnd.refract+json` content type. You may
+also send an [API Elements Parse
+Result](http://api-elements.readthedocs.io/en/latest/element-definitions/#parse-result-elements)
+as `application/vnd.refract.parse-result+json`.
+
+```
+application/vnd.refract+json
+application/vnd.refract.parse-result+json
```

Comment:
You also need to add a transaction example for the parse result too. We are using this APIB to do integration testing.